### PR TITLE
Add support for Aruba AOS-CX commands

### PR DESCRIPTION
**Pull request by scetron**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

aruba aoscx 

show vsf detail
show ntp associations
show mac-address-table
show lldp neighbors-info detail
show ip route all-vrfs
show interface dom detail
show interface
show bgp all-vrfs all summary
show bfd all-vrfs
show arp all-vrfs
show aaa authentication port-access interface all client-status

##### SUMMARY

Adding support for Aruba AOS-CX commands above. 

```

```

### CODE REVIEW

File: templates/index [hunk 1]
```diff
@@ -62,6 +62,18 @@ arista_eos_show_mlag.textfsm, .*, arista_eos, sh[[ow]] ml[[ag]]
 arista_eos_show_vlan.textfsm, .*, arista_eos, sh[[ow]] vl[[an]]
 arista_eos_show_vrf.textfsm, .*, arista_eos, sh[[ow]] vrf
 
+aruba_aoscx_show_aaa_authentication_port-access_interface_all_client-status.textfsm , .*, aruba_aoscx, sh[[ow]] aa[[a]] authe[[ntication]] port-access interface all client-status 
+aruba_aoscx_show_lldp_neighbors-info_detail.textfsm, .*, aruba_aoscx, sh[[ow]] ll[[dp]] nei[[ghbors-info]] d[[etail]]
+aruba_aoscx_show_interface_dom_detail.textfsm , .*, aruba_aoscx, sh[[ow]] int[[erface]] dom d[[etail]]
+aruba_aoscx_show_bgp_all-vrfs_all_summary.textfsm, .*, aruba_aoscx, sh[[ow]] bgp all-[[vrfs]] a[[ll]] s[[ummary]]
```
Comment (lvrfrc87) [inline templates/index:68]: @scetron this line needs to be moved at line 66

Comment (scetron) [reply templates/index:67]: Thanks!

**Comment by scetron** [comment]
Looks like all checks have passed! Let me know if there's anything else that needs adjusting.

**Comment by jvanderaa** [comment]
@scetron I think this may need a rebase. There are just a few too many files and commits involved here for just the Aruba AOS-CX commands.

**Pull request closed without merging**